lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -491,10 +491,10 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
   Widget _openCameraPickerAction(BuildContext context) {
     return SimpleContextMenuActionBuilder(
         Key('pick_from_camera_context_menu_action'),
-        SvgPicture.asset(imagePath.icPhotoLibrary,
+        SvgPicture.asset(imagePath.icCamera,
             width: 24, height: 24, fit: BoxFit.fill),
         //AppLocalizations.of(context).photos_and_videos
-        'Open camera' )
+        'Take a photo' )
```

Comment:
please use l10n to have localization.

Reply:
this is outdated its changed in final version